package.json

```diff
@@ -4,7 +4,10 @@
   "description": "An opinionated queue based worker server for node.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha -- --recursive --reporter dot test/unit",
+    "coverage-check": "istanbul check-coverage && echo 'Coverage check successful!'",
+    "lint": "eslint .",
+    "test": "mocha --recursive --reporter dot"
```

Comment:
Also there is a lot of log junk going on when testing, should we explicitly set the log level to none?